Multiplatform/NowPlaying/NowPlayingQueue.swift

```diff
@@ -95,7 +96,7 @@ struct NowPlayingQueue: View {
                 TipView(HistoryTip())
                     .listRowSeparator(.hidden)
                     .listRowBackground(Color.clear)
-                    .foregroundStyle(.black)
+                    .foregroundStyle(colorScheme == .dark ? .white : .black)
```

Comment:
Did you test this? Changing this does update the view for me.

Reply:
It does.

With this change:

<img width="392" alt="image" src="https://github.com/rasmuslos/AmpFin/assets/12995396/6813de3c-3f06-4426-89ad-29f9a18312e2">
<img width="371" alt="image" src="https://github.com/rasmuslos/AmpFin/assets/12995396/a8d99ea8-9ee6-4746-97a0-6dbda37a2d01">

Without this change:
<img width="389" alt="image" src="https://github.com/rasmuslos/AmpFin/assets/12995396/a91c81cc-35a2-49d6-a500-e90493b77e65">

Note the close button is much less visible on dark mode.